packages/teams-js/README.md

```diff
@@ -1,24 +1,12 @@
 # Microsoft Teams JavaScript client SDK
 
-Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch.
-
-This JavaScript library is part of the [Microsoft Teams developer platform](https://developer.microsoft.com/microsoft-teams/). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client).
-
-[![Build Status](https://travis-ci.org/OfficeDev/microsoft-teams-library-js.svg?branch=master)](https://travis-ci.org/OfficeDev/microsoft-teams-library-js)
-[![Coverage Status](https://coveralls.io/repos/github/OfficeDev/microsoft-teams-library-js/badge.svg?branch=master)](https://coveralls.io/github/OfficeDev/microsoft-teams-library-js?branch=master)
-
 ## Getting Started
 
-1.  Clone the repo
-2.  Navigate to the repo root and run `yarn install`
-3.  Run `yarn build` from repo root
-4.  To run Unit tests, run `yarn test`
-
-  TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the packages/teams-js directory.
+Whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the packages/teams-js directory.
```

Comment:
Can you add a sentence before this that links to the root README instructions for cloning the repo? Something like,

"See [instructions](./README.md#Getting-started) in the monorepo root for how to clone the repository."

Note: I haven't tested that link, so not sure if it's correct.

Reply:
I can't see your link but I know how to link to another page and section, so thank you !